src/tir/transforms/lower_sparse_buffer.cc

```diff
@@ -354,6 +354,76 @@ class BufferTransformer : public StmtExprMutator {
   bool is_horizontal_fuse_;
 };
 
+class Sparse_indices_buffer_post_process : public StmtExprMutator {
+  public:
+    explicit Sparse_indices_buffer_post_process(){}
+  private:
+    Stmt VisitStmt_(const BlockNode* op) final{
+      if(support::StartsWith(op->name_hint,"binary_search_block"))
```

Comment:
Please format the code with [clangformat](https://clang.llvm.org/docs/ClangFormat.html) (use the default LLVM style is fine), otherwise you will not pass the CI.

Reply:
We can add some block annotations such as `{"is_binary_search_block": true}` when emitting binary search blocks, and check the annotations here instead of using block name.